<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="16" column="1">
     <widget class="QPushButton" name="FFTButton">
      <property name="text">
       <string>Toggle Fast Fourier Transform (FFT)</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="PlotWidget" name="graphWidget" native="true"/>
    </item>
    <item row="14" column="1">
     <widget class="QPushButton" name="playButton">
      <property name="text">
       <string>Pause/Play</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QPushButton" name="zoomoutButton">
      <property name="text">
       <string>Zoom Out</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QPushButton" name="zoominButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Zoom In</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
